admin/data-collection.rst

```diff
@@ -184,14 +159,14 @@ Source node override
 ~~~~~~~~~~~~~~~~~~~~
 
 Source node of metrics collection. This can be used when other node provides
-information about this node. In this way collected data can be collected and
-shown on right nodes.
+information about local node. In this way, platform provides additional flexability of where 
+metrics collection is taking place.
 
-Other example of usage is virtual nodes (nodes with IP 0.0.0.0). In this case
-node state can be obtained from the DCI created on this node but collected
+Other example of usage is virtual nodes (nodes with IP 0.0.0.0). In this case,
+node state can be obtained from the DCI created on local node, but collected
 from the other one.
 
-Data is collected from the same node if no value set.
+Data is collected from the local node if no value is set.
 
 Polling
```

Comment:
If we will check screenshot Polling is called part with Data type, units and multuplier option. Not part about how often to poll. So this part should be renamed as "Collection schedule" and "Polling" part need to be added movind inside Data type part